openslides/core/static/js/core/start.js

```diff
@@ -46,6 +47,27 @@ angular.module('OpenSlidesApp.core.start', [])
                 this.bootup();
             },
         };
+
+        // We need to 'ping' the server with a get request to whoami, because then we can decide,
+        // if the server is down or respond with a 403 (this cannot be differentiated with websockets)
+        autoupdate.registerRetryConnectCallback(function () {
+            return $q(function (resolve) {
```

Comment:
I think this extra promise envelop is not necessary. $http.get().then() returns a promise. Just just have to return true or false in the success and error callback.